### ISSUE TITLE
Export Project window's Path can not be changed to other directories of the same drive

### ISSUE THREAD

**Issue by onigumo69** [godotengine/godot#48317]
**Godot version:**
3.3

**OS/device including version:**
Windows 10 1909

**Issue description:**
Export Project window's Path can not be changed by clicking on the drive selection if the actual project is in that drive.

**Steps to reproduce:**
Project -> Export... -> Export Project -> Choose different hard drives -> the one's path has the actual project is fixed
( For example in my case I have C D E drives, and my actual Godot project is in D, when I click C or E it will change the path into their root directory, but when I click D, it's fixed in my actual project path which is D:\ass\cursor\ , and can only be changed if I manually type in the address bar )

**Minimal reproduction project:**

![1619751085(1)](https://user-images.githubusercontent.com/46274753/116642694-0a726f80-a9a2-11eb-9904-a79dd47e4e41.jpg)

**Comment by akien-mga** [comment]
Maybe it's just that it remembers your current location for each drive? You can try to browse to a specific folder on `C:`, switch to `D:`, then switch back to `C:` and see if you're in `C:\` or in that specific folder.

**Comment by onigumo69** [comment]
> Maybe it's just that it remembers your current location for each drive? You can try to browse to a specific folder on `C:`, switch to `D:`, then switch back to `C:` and see if you're in `C:\` or in that specific folder.

nope, all other drives will still be switch to the root directory after they had switched to a specific folder before.
